Gopkg.toml

```diff
@@ -7,12 +7,14 @@ ignored = ["google.golang.org/appengine*"]
   non-go = true
 
 [[constraint]]
-  branch = "master"
+  branch = "git-hook"
   name = "code.gitea.io/git"
+    source = "https://github.com/codeocean/gitea-git.git"
```

Comment:
Why change this?

Reply:
This is temporary until the pull requests this depends on are merged so that this builds in CI. This needs to be removed and dep updated to point back to master before merging, as I said in the PR description.